lib/membrane/pipeline.ex

```diff
@@ -331,7 +331,7 @@ defmodule Membrane.Pipeline do
         A proxy for `#{inspect(unquote(__MODULE__))}.start_link/3`
         """
         @spec start_link(
-                pipeline_options :: Pipeline.pipeline_options_t(),
+                pipeline_options :: Membrane.Pipeline.pipeline_options_t(),
```

Comment:
```suggestion
                pipeline_options :: unquote(__MODULE__).pipeline_options_t(),
```

not to make this mistake again ;)